script/validate-competencies-json.js

```diff
@@ -33,9 +35,18 @@ validateJsonFiles([
 async function validateJsonFiles(configurations) {
 	let errors = [];
 	for (const configuration of configurations) {
-		errors = errors.concat(
-			await validateJsonFileUsingSchema(configuration.path, configuration.schema)
-		);
+		try {
+			errors = errors.concat(
+				await validateJsonFileUsingSchema(configuration.path, require(configuration.schema))
```

Comment:
The reason we're using a `try/catch` here is because we now need to `require` each schema at the time where it's tested. Now that the schemas are written in JavaScript and have some logic of their own, it's possible for loading of them to fail, e.g. if there's an error in the `levels` JSON file then this will prevent the competencies schema from even loading.

We now `require` each schema as late as possible, they are executed in order so if there's an issue with the `levels` or `domains` JSON files then those errors will be surfaced before trying to load in the competencies schema which depends on them. Without this, errors in the `levels` or `domains` JSON files would be much harder to debug.